src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -182,6 +182,25 @@ internal class RecommendationService(
     }
   }
 
+  private fun updatePageReviewedValues(
+    updateRecommendationRequest: RecommendationModel,
+    recommendationEntity: RecommendationEntity
+  ): RecommendationEntity {
+    if (updateRecommendationRequest.hasBeenReviewed?.personOnProbation == true) {
+      val data = recommendationEntity.data
+      val personOnProbation = data.personOnProbation
+      return recommendationEntity.copy(
+        data = data.copy(
+          personOnProbation = personOnProbation?.copy(
+            hasBeenReviewed = true
+          ),
+          hasBeenReviewed = null
```

Comment:
Should it default to `false`?

Reply:
I copy Jon's comment :)

Reply:
This should clear the whole 'hasBeenReviewed ' object from the database after each request from the frontend. We don't really want to store this field in 2 different places in the database as could get confusing, hence the reason for setting to null. The 'hasBeenReviewed' under 'personOnProbation' object will only be true or false as stated in ticket.